homeassistant/components/sensor/sonarr.py

```diff
@@ -96,16 +97,24 @@ def __init__(self, hass, conf, sensor_type):
         self._icon = SENSOR_TYPES[self.type][2]
 
         # Update sensor
+        self._available = False
         self.update()
 
     def update(self):
         """Update the data for the sensor."""
         start = get_date(self._tz)
         end = get_date(self._tz, self.days)
-        res = requests.get(
-            ENDPOINTS[self.type].format(
-                self.ssl, self.host, self.port, self.apikey, start, end),
-            timeout=5)
+        try:
+            res = requests.get(
+                ENDPOINTS[self.type].format(
+                    self.ssl, self.host, self.port, self.apikey, start, end),
+                timeout=5)
+        except OSError:
+            _LOGGER.error('Host %s is not available', self.host)
+            self._available = False
+            self._state = STATE_UNAVAILABLE
```

Comment:
I see now that you already leverage the property, you can just set `self._state` to `None`